### PR TITLE
add Learning software

### PR DESCRIPTION
**Pull request by libialany**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/scholarsome.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+platforms:
+  - Node
+  - Docker
+tags:
+  - Learning and Courses
+source_code_url: https://github.com/hwgilbert16/scholarsome
+demo_url: https://scholarsome.com/
+stargazers_count: 505
+updated_at: '2024-07-23'
+archived: false
```
Comment (kokomo123) [inline software/scholarsome.yml:15]:
```suggestion
```
these are redundant lines, they will be automatically maintained later 👍🏻

Comment (libialany) [reply software/scholarsome.yml]: Thank you for the advice, I fix that

File: software/scholarsome.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: scholarsome
+website_url: https://www.scholarsome.com/
+description: Web-based and open source interactive flashcard learning software studying for the masses.
+licenses:
+  - GPL-3.0
+platforms:
+  - Node
```
Comment (Ki-er) [inline software/scholarsome.yml:7]:
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
https://scholarsome.com/handbook/installation/installing/
Since development only uses NodeJS rather than Deployment it can be removed

**Review by nodiscc — APPROVED** [review]

**Pull request merged**